x-pack/plugins/observability_solution/observability_ai_assistant/server/utils/recall/score_suggestions.ts

```diff
@@ -127,15 +121,9 @@ export async function scoreSuggestions({
     scoreFunctionRequest.message.function_call.arguments
   );
 
-  const scores = parseSuggestionScores(scoresAsString).map(({ id, score }) => {
-    const originalSuggestion = suggestionsWithShortId.find(
-      (suggestion) => suggestion.shortId === id
-    );
-    return {
-      originalId: originalSuggestion?.originalId,
-      score,
-    };
-  });
+  const scores = parseSuggestionScores(scoresAsString)
+    // Restore original IDs
+    .map(({ id, score }) => ({ id: shortIdTable.lookup(id)!, score }));
```

Comment:
Why were we not using the `lookup` method before? Seems much simpler but is there something I'm missing?